### PR TITLE
fix: make possible to have white spaces in root path

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: bin/emqx [hunk 1]
```diff
@@ -143,11 +143,7 @@ if [ "$ES" -ne 0 ]; then
     exit $ES
 fi
 
-if [ -z "$WITH_EPMD" ]; then
-    EPMD_ARG="-start_epmd false -epmd_module ekka_epmd -proto_dist ekka"
-else
-    EPMD_ARG="-start_epmd true $PROTO_DIST_ARG"
```
Comment (thalesmg) [inline bin/emqx:149]: Q: is this removal intentional?  I thought that only in 5.0 we dropped support for epmd.

**Review by thalesmg — APPROVED** [review]

File: bin/emqx (hunk 1 above)
Comment (zmstone) [reply bin/emqx:149]: we never really supported it since 4.2

Comment (zmstone) [reply bin/emqx:149]: restored the support. 
just to keep it compatible.

File: bin/emqx [hunk 2]
```diff
@@ -60,62 +66,157 @@ fi
...
+        echo "Usge $REL_NAME rpcterms MODULE FUNCTION [ARGS, ...]"
+        echo "Connect to the EMQX node and make an Erlang RPC"
+        echo "The result of the RPC call is pretty-printed as an "
+        echo "Erlang term"
+        ;;
+    root_dir)
+        echo "Print EMQX installation root dir"
+        ;;
+    eval)
+        echo "Evaluate an Erlang or Elxir expression in the EMQX node"
```
Comment (thalesmg) [inline bin/emqx:133]: Nit: for 4.3, there's no Elixir.

File: bin/emqx [hunk 3]
```diff
@@ -372,9 +457,10 @@ if [ -z "$NAME_ARG" ]; then
         if [ "$IS_BOOT_COMMAND" = 'no' ]; then
             # for non-boot commands, inspect vm.<time>.args for node name
             # shellcheck disable=SC2012,SC2086
-            LATEST_VM_ARGS="$(ls -t $RUNNER_DATA_DIR/configs/vm.*.args | head -1)"
+            LATEST_VM_ARGS="$(ls -t $RUNNER_DATA_DIR/configs/vm.*.args 2>/dev/null | head -1)"
             if [ -z "$LATEST_VM_ARGS" ]; then
-                echo "For command $1, there is no vm.*.args config file found in $RUNNER_DATA_DIR/configs/"
+                echoerr "For command $1, there is no vm.*.args config file found in $RUNNER_DATA_DIR/configs/"
+                echoerr "Please mkae sure the node is initialized (sarted for at least onces)"
```
Comment (thalesmg) [inline bin/emqx:463]:
```suggestion
                echoerr "Please make sure the node is initialized (started for at least once)"
```

**Review by thalesmg — APPROVED** [review]

**Review by terry-xiaoyu — APPROVED** [review]
